crates/flox/src/commands/search.rs

```diff
@@ -197,6 +202,16 @@ fn render_search_results_user_facing(
         writeln!(&mut writer, "{package:<column_width$}  {desc}")?;
     }
     writer.flush().context("couldn't flush search results")?;
+    if let Some(count) = search_results.count {
+        // Don't show the message if we have exactly the number of results as the limit,
+        // otherwise we would get messages like `Showing 10 of 10...`
+        if count != n_results as u64 {
+            eprint!(
+            "\nShowing {} of {} results. Use `flox search {{query}} --all` to see the full list.",
+            *SEARCH_LIMIT, count
+        );
+        }
+    }
     eprintln!("\nUse `flox show {{package}}` to see available versions");
```

Comment:
```suggestion
    info!("Use `flox show {{package}}` to see available versions");
```

Reply:
Do these `info!` macros print to `stderr`?

Reply:
yes

Reply:
Fixed